src/solidity-frontend/solidity_convert.h

```diff
@@ -186,6 +186,248 @@ class solidity_convertert
   bool get_elementary_type_name_int(
     SolidityGrammar::ElementaryTypeNameT &type,
     typet &out);
+  
+  /*
```

Comment:
There's no implementation for the method declarations below. please remove if not needed.

Reply:
No problem.

Reply:
Hi, I think the implementation can be found at solidity_convert.cpp:2074. It's used in the get_elementary_type() function. So I'll keep it.

Reply:
I think he means the code changes in lines 189 to 430.

Reply:
Noted, thanks!